ansible/app.yml

```diff
@@ -1,18 +1,18 @@
-- hosts: all
-  become: true
-  roles:
-    - aeriscloud.wkhtmltopdf
-    - geerlingguy.mysql
-    - mysql
-
-- hosts: all
-  become: yes
-  roles:
-      - { role: common }
```

Comment:
````
---

- name: Install packages
  apt: pkg={{ item }} state=latest
  with_items:
      - htop
      - python-mysqldb

- name: Accept github hostkey because other solution didn't seems to work
  shell: ssh-keyscan -H github.com >> /etc/ssh/ssh_known_hosts

- name: cache symfony 2
  file: >
    path=/var/cache/symfony
    state=directory
  become: yes

- name: logs symfony 2
  file: >
    path=/var/logs/symfony
    state=directory
  become: yes

- name: acl symfony 2
  shell: >
    setfacl -R -m u:www-data:rwX -m u:app:rwX /var/cache/symfony /var/logs/symfony
  become: yes

- name: acl default symfony 2
  shell: >
    setfacl -dR -m u:www-data:rwX -m u:app:rwX /var/cache/symfony /var/logs/symfony
  become: yes
```